README.md

```diff
@@ -195,4 +195,140 @@ amtHexString := amount.ToHexString() // type `string`
 ### `MAX_BALANCES` and `MAX_ORDERS`
 Because our system encodes all its computation in zero-knowledge "circuits", the size of each wallet must be known ahead of time. To this end, we impose the restriction that each wallet has at most `MAX_BALANCES = 10` balances, and `MAX_ORDERS = 4` orders. 
 
-The SDK and the relayer will prevent you from allocating more balances and orders than are allowed.
\ No newline at end of file
+The SDK and the relayer will prevent you from allocating more balances and orders than are allowed.
+
+# External (Atomic) Matching
+We also allow for matches to be generated _externally_; meaning generated as a match between a Renegade user -- with state committed into the darkpool -- and an external user, with no state in the darkpool.
+
+To generate an external match, a client may request an `ExternalMatchBundle` from the relayer. This type contains:
+- The result of the match, including the amount and mint (erc20 address) of each token in the match
+- A transaction that the client can submit to the protocol to settle the match.
```

Comment:
"submit to the chain" is more correct than "submit to the protocol" here

Reply:
also may be helpful to include somewhere that partial fills are possible, in case readers are not familiar with the rest of the protocol